src/searching.jl

```diff
@@ -0,0 +1,41 @@
+using GeometryBasics
+using Delaunator
+
+
+function _prevedge(i::Integer)
+    (i-1) % 3 == 0 ? i+2 : i -1
+end 
+
+function findtriangle(t, p)
+  current = 1
+  start = current
+  n = 0
+
+  while true
+      next = Delaunator._nextedge(current)
```

Comment:
You should be able to drop the Delaunator. prefix since it's all in the same module.